### PR TITLE
Bump up httpclient dependency due to whitesource vulnerability findin…

### PR DESCRIPTION
**Pull request by sfc-gh-japatel**
…g on older version

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -225,7 +225,7 @@
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>4.5.2</version>
+            <version>4.5.13</version>
```
Comment (sfc-gh-giyer) [inline pom.xml:228]: how did you find the version that we should be using?

Comment (sfc-gh-japatel) [reply pom.xml:228]: Few ways. 
1. Latest version in maven. 
2. JDBC also bumped up to this version. 
3. Succesful test on PR: https://travis-ci.org/github/snowflakedb/snowflake-ingest-java/builds/744319475

Comment (sfc-gh-japatel) [reply pom.xml:228]: Checked with whitesource software. version 4.5.13 fixes the mentioned vulnerability and is approved by it.

**Comment by sfc-gh-psaraiya** [comment]
What vulnerability are we trying to fix?

**Comment by sfc-gh-japatel** [comment]
> What vulnerability are we trying to fix?

https://snowflakecomputing.atlassian.net/browse/SNOW-170633
Vul: https://github.com/apache/httpcomponents-client/commit/0554271750599756d4946c0d7ba43d04b1a7b220

File: pom.xml (hunk 1 above)
Comment (sfc-gh-japatel) [reply pom.xml:228]: @sfc-gh-giyer can I push this?

**Review by sfc-gh-jliang — APPROVED** [review]
lgtm

**Pull request merged**